lib/sisimai/bite/json/amazonses.rb

```diff
@@ -0,0 +1,255 @@
+module Sisimai::Bite::JSON
+  # Sisimai::Bite::JSON::AmazonSES parses a bounce object(JSON) which created
+  # by Amazon Simple Email Service.
+  # Methods in the module are called from only Sisimai::Message.
+  module AmazonSES
+    # Imported from p5-Sisimail/lib/Sisimai/Bite/JSON/AmazonSES.pm
+    class << self
+      require 'sisimai/bite/json'
+
+      # https://docs.aws.amazon.com/en_us/ses/latest/DeveloperGuide/notification-contents.html
+      BounceType = {
+        :Permanent => {
+          :General    => '',
+          :NoEmail    => '',
+          :Suppressed => '',
+        },
+        :Transient => {
+          :General            => '',
+          :MailboxFull        => 'mailboxfull',
+          :MessageTooLarge    => 'mesgtoobig',
+          :ContentRejected    => '',
+          :AttachmentRejected => '',
+        },
+      }
+
+      def smtpagent;   return Sisimai::Bite.smtpagent(self); end
+      def description; return 'Amazon SES(JSON): http://aws.amazon.com/ses/'; end
+
+      # Parse bounce messages from Amazon SES(JSON)
+      # @param         [Hash] mhead       Message headers of a bounce email
```

Comment:
__[rubocop]__
* Style/Semicolon: Do not use semicolons to terminate expressions. [:ref](https://github.com/bbatsov/ruby-style-guide#no-semicolon)
* Layout/TrailingWhitespace: Trailing whitespace detected. [:ref](https://github.com/bbatsov/ruby-style-guide#no-trailing-whitespace)

<!-- Comment by [SideCI](https://sideci.com) -->